docs/Guides/Detecting-When-Clients-Abort.md

```diff
@@ -55,7 +55,7 @@ const app = Fastify({
 
 app.addHook('onRequest', async (request, reply) => {
   request.raw.on('close', () => {
-    if (request.raw.aborted) {
+    if (request.raw.destroyed) {
```

Comment:
I bit late for the comment, but `destroyed` is not identical to `aborted`.
`destoryed` should always `true` when calling `request.destroy()`. However, it doesn't indicate the `request` is `aborted` (meaning `readableEnded` is not `true` or `complete` is not `true`.)

https://github.com/nodejs/node/blob/f9737b1b8c96e43cedfff764634524df62cf3109/lib/_http_incoming.js#L222-225

Reply:
It seems is deprecated but still we use it for the `onRequestAbort` hook ([ref](https://github.com/fastify/fastify/blob/cc347d7c0b4266097b61b126158b797878668353/lib/route.js#L495))

Maybe we should change the docs to `onRequestAbort` as best-effort?

Reply:
> It seems is deprecated but still we use it for the onRequestAbort hook

Yes, it is deprecated and we are still using.
But I assume it would never been able to remove.

> Maybe we should change the docs to onRequestAbort as best-effort?

The guide is explaining request abort in detail. I would keep it since the function is reference this document for further explanation.